ccm_web/server/src/canvas/canvas.interfaces.ts

```diff
@@ -114,6 +159,18 @@ export interface CanvasEnrollment {
   type: UserEnrollmentType
 }
 
+export interface CanvasUser {
+  id?: number
+  name: string
+  sortable_name: string
+  short_name?: string
```

Comment:
In some circumstances the Canvas API wouldn't return an `id` or `short_name`.  Mostly, I think it's just after a user has been created.  I thought it'd be useful to use this same interface to define the results of creating a user as well as the results of looking up a user.

Reply:
Yes, this is probably fine, since we don't really use these values in any substantive way. I'm a little surprised `id` is not always present though. If we planned to use the data consistently, it may make sense to create two interfaces just so we could more accurately type the expected return object.

Reply:
Last week before you made changes to the response structure, I had to "stuff" the `login_id` (or maybe it was `email`) into the responses from the enrollment step, because those used only `id`.  When assembling the response organized by user email, I couldn't correlate the enrollment results with the creation results because the creations didn't include `id`.

Reply:
🏁

Reply:
I'm seeing that `id` and `short_name` are always present in the responses for the two endpoints you added. We can make them not optional.

Reply:
Actually, `login_id` also seems to be present in both cases. Could we just add `login_id` to `CanvasUser`, then delete `CanvasUserLoginEmail` and replace all references to it with `CanvasUser`?

Reply:
We resolved this.